pages/index.html

```diff
@@ -42,7 +42,7 @@ <h2>Self hostable</h2>
 	<p class="left">You are convinced by FreshRSS but you do not know what you need? Keep calm and have a look at the following table:</p>
 
 	<table class="alternate striped-vert">
-		<caption>Software basis for FreshRSS</caption>
+		<caption>Software basis for FreshRSS for 1.7 release</caption>
```

Comment:
Maybe `FreshRSS 1.10 requirements`? I think a software basis is something slightly different than intended (e.g., GNU Core Utilities is a common software basis for most distros).